terraform-build-user/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-build-user.tfstate"
```

Comment:
This key needs to be updated.

Reply:
Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/29fe68a31f29ec3fc4578fbba4c10567901424a8.